HOWTO.md

```diff
@@ -0,0 +1,10 @@
+# How to experiment ComputePressure API with Chrome
+
+The ComputePressure API is currently available only in Chrome Canary behind a flag.
+
+1) Download a version of Chrome with ComputePressure implemented ([Desktop](https://www.google.com/intl/en_ie/chrome/canary/)).
+2) Launch Chrome with the command line flag --enable-features=ComputePressure.
```

Comment:
On Windows this is more complicated

Right click on Chrome Canary icon

![image](https://user-images.githubusercontent.com/1101339/207826604-7501ad40-bdf2-4d3e-a0ff-ffe9bc76aa9a.png)


Triple right click on google Chrome Canary:

![image](https://user-images.githubusercontent.com/1101339/207826847-53e9a61e-061c-4a38-b05a-4e74f9d4277d.png)

Choose Properties

Add --enable-features=ComputePressure at the end in Target: (after the last ")

![image](https://user-images.githubusercontent.com/1101339/207827028-2eca258a-03fc-4b80-83f0-55989735f8d9.png)